server/auth/types/authentication_type.ts

```diff
@@ -223,6 +224,11 @@ export abstract class AuthenticationType implements IAuthenticationType {
     return selectedTenant;
   }
 
+  isPageRequest(request: KibanaRequest) {
+    const path = request.url.pathname || '/';
+    return path.startsWith('/app/') || path === '/' || path.startsWith('/goto/');
```

Comment:
What does `/goto` mean?

Reply:
/goto is the path of saved short URLs prefix